meson.build

```diff
@@ -0,0 +1,79 @@
+project(
+    'minmea',
+    'c',
+    default_options: ['c_std=c99', 'buildtype=debug'],
+)
+
+add_project_arguments(
+    '-Wall', '-Wextra', '-Werror',
+    '-D_POSIX_C_SOURCE=199309L', '-D_BSD_SOURCE', '-D_DEFAULT_SOURCE',
+    '-D_DARWIN_C_SOURCE',
+    language: 'c',
+)
+
+if host_machine.system() == 'windows'
+    add_project_arguments('DMINMEA_INCLUDE_COMPAT')
+endif
+
+c_args = []
+
+if get_option('buildtype') == 'debug'
+    c_args += '-ggdb'
+endif
+
+minmea_includes = include_directories('.')
+
+minmea_sources = ['minmea/minmea.c']
+
+minmea_dependencies = []
+
+minmea_lib = library(
+    'minmea',
+    minmea_sources,
+    include_directories: minmea_includes,
+    dependencies: minmea_dependencies,
```

Comment:
This is an empty array, is it just future proofing? The project should confirm whether it's worth the additional lines for something that may have a definite development policy to never utilize.

In fact this seems likely to be the case, the project goals state that it is lightweight/minimal, and mention the bullet point: 
- One source file and one header - can't get any simpler.

So it sounds like they would not be willing to add dependencies in the future, which means you don't need to plan ahead for dependencies.